.github/workflows/api-tests.yml

```
@@ -46,3 +46,10 @@ jobs:
     - name: Stop containers
       if: always()
       run: docker compose down
+
+  mark-tests-passed-if-no-changes:
+    if: needs.changes-check.outputs.any_changed != 'true'
```

actionlint

property "changes-check" is not defined in object type {}

(expression)